libstorage/RocksDBStorage.cpp

```diff
@@ -0,0 +1,220 @@
+/*
+ * @CopyRight:
+ * FISCO-BCOS is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * FISCO-BCOS is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with FISCO-BCOS.  If not, see <http://www.gnu.org/licenses/>
+ * (c) 2016-2018 fisco-dev contributors.
+ */
+/** @file RocksDBStorage.cpp
+ *  @author bxq
+ *  @date 20180423
+ */
+
+#include "RocksDBStorage.h"
+#include "StorageException.h"
+#include "Table.h"
+#include "rocksdb/db.h"
+#include "rocksdb/options.h"
+#include "rocksdb/slice.h"
+#include "rocksdb/write_batch.h"
+#include <libdevcore/Common.h>
+#include <libdevcore/FixedHash.h>
+#include <libdevcore/Guards.h>
+#include <libdevcore/RLP.h>
+#include <libdevcore/easylog.h>
+#include <tbb/parallel_for.h>
+#include <memory>
+#include <thread>
+
+using namespace dev;
+using namespace dev::storage;
+using namespace rocksdb;
+
+Entries::Ptr RocksDBStorage::select(
+    h256, int, TableInfo::Ptr tableInfo, const std::string& key, Condition::Ptr condition)
+{
+    try
```

Comment:
Don't need to callback condition firstly as LevelDBStorage?

Reply:
condition is used in leveldbstorage2/sqlstorage/rocksdbstorage